src/Stepping.TmProviders.LocalTm/Stepping/TmProviders/LocalTm/Models/TmTransactionModel.cs

```diff
@@ -0,0 +1,54 @@
+﻿using Stepping.Core.Databases;
+using Stepping.TmProviders.LocalTm.Steps;
+
+namespace Stepping.TmProviders.LocalTm.Models;
+
+public class TmTransactionModel
+{
+    public string Gid { get; set; } = null!;
+
+    public string Status { get; set; } = null!;
+
+    public LocalTmStepModel Steps { get; set; } = null!;
+
+    public DateTime CreateTime { get; set; }
```

Comment:
It should be `CreationTime` since the `Create` is a verb.

Refer to https://helpx.adobe.com/coldfusion/cfml-reference/coldfusion-functions/functions-c-d/CreateTime.html